modules/3d/src/usac/ransac_solvers.cpp

```diff
@@ -372,8 +372,8 @@ void UniversalRANSAC::initialize (int state, Ptr<MinimalSolver> &min_solver, Ptr
     } else if (params->isPnP()) {
         degeneracy = makePtr<Degeneracy>();
         if (min_sample_size == 3) {
-            min_solver = P3PSolver::create(points, calib_points, K1);
-            non_min_solver = DLSPnP::create(points, calib_points, K1);
+            min_solver = P3PSolver::create(points, calib_points, K1);   // -->
```

Comment:
Nitpicking.
Another P3P implementation. Would have been great to reuse code from `SOLVEPNP_AP3P` for instance.
Or made this implementation code available in the lib.